src/devices/vdml_rotation.cpp

```diff
@@ -13,6 +13,11 @@
 #include "pros/rotation.hpp"
 
 namespace pros {
+    
+Rotation::Rotation(const std::uint8_t port, const bool reverse_flag) : _port(port) {
+    pros::c::rotation_init_reverse(port, reverse_flag);
```

Comment:
minor task but it should be done, could we fix the spaces vs tabs issue with everything in this file?